app/main.py

```diff
@@ -1 +1,40 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str, appetite: int, is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
```

Comment:
According to the checklist, it's recommended to use `@staticmethod` for some functions. In this case, the `print_name` method could be a static method, as it doesn't use any instance variables or methods. Here's how you can improve it:

```python
@staticmethod
def print_name(name):
    print(f"Hello, I'm {name}")
```